css/all.css

```diff
@@ -5,6 +5,8 @@
     --hover-color: hsl(var(--color-hue), 55%, 40%);
     --link-color: var(--hover-color);
     --border-color: hsl(var(--color-hue), 60%, 25%);
+    --color-on-background: white;

```

Comment:
Can you change the name of this variable to `--contrast-text-color`? It's more in line with our modern engine naming convention.